crates/api_models/src/payment_methods.rs

```diff
@@ -166,6 +166,7 @@ pub struct CardDetailsPaymentMethod {
 pub struct PaymentMethodDataBankCreds {
     pub mask: String,
     pub hash: String,
+    pub mca_id: String,
```

Comment:
This needs to be in the `BankAccountConnectorDetails` type, as two entries in the `PaymntMethodDataBankCreds.connector_details` vector may have different MCA IDs.